infra-as-code/bicep/modules/logging/README.md

```diff
@@ -109,35 +124,55 @@ az deployment group create \
 # Set Platform management subscripion ID as the the current subscription
 $ManagementSubscriptionId = "[your platform management subscription ID]"
 
+# Set the top level MG Prefix in accordance to your environment. This example assumes default 'alz'.
+$TopLevelMGPrefix = "alz"
+
+$ResourceGroupName = "rg-$TopLevelMGPrefix-logging-001"
```

Comment:
Why separated instead of concatenated like the Bash example above?

Reply:
Same as the other bash comments, I won't flood your inbox!